lib/ssh_scan/policy.rb

```diff
@@ -25,5 +26,20 @@ def self.from_string(string)
       opts = YAML.load(string)
       self.new(opts)
     end
+
+    # Choose a policy file based on rules
+    def self.first_match(result)
+      Dir.glob("policies/**/*.yml") do |policy_file|
```

Comment:
My understanding would be that it will work when you run from source, but not when you run as a gem.

You can test this by doing the following:

1.) gem build ssh_scan.gemspec # build a gem
2.) gem install ssh_scan-0.0.16.gem # install the gem
3.) go to a different directory and run to exercise this code in IRB and see if the path still works